src/Helpers/Configuration.ts

```diff
@@ -0,0 +1,11 @@
+import { keys, fromPairs, map, pipe } from 'ramda'
+
+import { camelCaseToScreamingSnakeCase } from './camelCaseToScreamingSnakeCase'
+
+const toPair = (s: string) => [camelCaseToScreamingSnakeCase(s), s]
+
+export const createEnvToConfigurationKeyMap = pipe(
```

Comment:
Since this function only cares about the values of the passed object, maybe it's best for it to actually receive the `keys` already, and let the caller have the responsibility (and freedom) of choosing _how_ to get those keys.

Also: since we don't have good typings for `pipe`, can you add a type signature to `createEnvToConfigurationKeyMap` indicating input and output type?

Reply:
Ok cool. I've updated everything.